shared/liboxide/devicesettings.cpp

```diff
@@ -103,6 +103,11 @@ namespace Oxide {
             _deviceType = DeviceType::RM2;
             return;
         }
+        if (modelName.contains("reMarkable Ferrari")) {
```

Comment:
This should be startswith as `Not reMarkable Ferrari` should not match. You could also do direct comparisons assuming there are no newlines in the value.